py-scoring/src/ai/h2o/sparkling/ml/params/H2OTargetEncoderMOJOParams.py

```diff
@@ -92,6 +92,16 @@ class H2OTargetEncoderMOJOParams(Params):
         "A seed of the generator producing the random noise",
         H2OTypeConverters.toInt())
 
+    problemType = Param(
+        Params._dummy(),
+        "problemType",
+        """A type of ML problem type for which the target encoder will be used for:
+              Auto           - If this option is chosen (default), the problem type will be automatically resolved based
```

Comment:
Maybe it should specify what is the logic behind it.

Eg. some people might expect 0/1 column to be categorical.

Reply:
Good point, i will also make the logic simpler. ( boolean or string => classification, otherwise regression)